src/coreclr/System.Private.CoreLib/src/System/GC.CoreCLR.cs

```diff
@@ -23,7 +23,8 @@ public enum GCCollectionMode
     {
         Default = 0,
         Forced = 1,
-        Optimized = 2
+        Optimized = 2,
+        Aggressive = 3,
```

Comment:
We seed the documentation for new APIs with XML comments.  Can you add XML comments for the new enum?  You can fill in the other comments while you're at it, e.g.
```C#
/// <summary>Specifies the behavior for a forced garbage collection.</summary>
public enum GCCollectionMode
{
    /// <summary>The default setting for this enumeration, which is currently <see cref="GCCollectionMode.Forced" />.</summary>
    Default = 0,

    /// <summary>Forces the garbage collection to occur immediately.</summary>
    Forced = 1,

    /// <summary>Allows the garbage collector to determine whether the current time is optimal to reclaim objects.</summary>
    Optimized = 2,

    /// <summary>NEW DESCRIPTION GOES HERE</summary>
    Aggressive = 3,
}
```